compiler-rt/lib/asan/asan_malloc_win.cpp

```diff
@@ -322,6 +322,22 @@ void *SharedReAlloc(ReAllocFunction reallocFunc, SizeFunction heapSizeFunc,
       }
     }
 
+    if (dwFlags & HEAP_REALLOC_IN_PLACE_ONLY) {
```

Comment:
What's the reason for doing the check here, instead of on line 367 (which would become redundant)?